src/__tests__/class-list.ts

```diff
@@ -0,0 +1,167 @@
+import { add, remove, toggle } from "../class-list";
+
+describe("classList", () => {
+  let element;
```

Comment:
this one change and the whole thing is strict compliant

```suggestion
  let element: HTMLDivElement;
```